app/code/Magento/Bundle/view/adminhtml/templates/catalog/product/edit/tab/attributes/extend.phtml

```diff
@@ -3,35 +3,33 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```

Comment:
I believe that no make sense use it in phtml files.

Reply:
Why not?

Reply:
We don't use any type declarations in phtml files. phtml files are representation and should not contain any logic and should be refactored if you find such of them

Reply:
Are you going to block this PR if these changes are not made?